test/helpers/create-adjustment-payment-request.js

```diff
@@ -13,7 +14,11 @@ const getValue = (currentValue, adjustmentType) => {
     return currentValue + 50
   }
   if (adjustmentType === RECOVERY) {
-    return currentValue - 50
+    if (processingConfig.handleSchemeClosures === true) {
+      return -currentValue
+    } else {
+      return currentValue - 50
+    }
```

Comment:
I don't think we should change this here as it may prevent existing tests working as expected as they're behaviour is now tied to the whatever the config value is set as.

I'd suggest adding new tests to the `process-payment-requests.test.js` but change the value of the `handleSchemeClosures` to confirm that if `handleSchemeClosures` is false, then we don't get any AR requests in the `completedPaymentRequests` property when we usually would.

Also a test to make sure nothing is routed to the request editor in that scenario.

That should be sufficient to cover the behaviour intended by this change

Reply:
tests added and changes to this file reverted